python/cuxfilter/charts/core/aggregate/core_aggregate.py

```diff
@@ -7,6 +7,20 @@ class BaseAggregateChart(BaseChart):
     use_data_tiles = True
     custom_binning = False
 
+    def _compute_array_all_bins(
+        self, source_x, source_y, update_data_x, update_data_y
+    ):
+        """
+        source_x: current_source_x, np.array()
+        source_y: current_source_y, np.array()
+        update_data_x: updated_data_x, np.array()
+        update_data_y: updated_data_x, np.array()
+        """
+        result_array = np.zeros(shape=(int(source_x.max()),))
+        # -1 for 0-based indexing
+        np.put(result_array, update_data_x.astype("int") - 1, update_data_y)
```

Comment:
nit: this (and others) could just be `astype(int)` which is slightly preferable over magic strings

Reply:
fixed